.azure/pipelines/jobs/default-build.yml

```diff
@@ -120,16 +120,10 @@ jobs:
       ${{ if eq(parameters.agentOs, 'Windows') }}:
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           name: NetCore1ESPool-Public
-          ${{ if ne(parameters.isAzDOTestingJob, true) }}:
-            # Visual Studio Build Tools
-            demands: ImageOverride -equals Build.Server.Amd64.VS2019.BT.Open
-          ${{ if eq(parameters.isAzDOTestingJob, true) }}:
-            # Visual Studio Enterprise - contains some stuff, like SQL Server and IIS Express, that we use for testing
-            demands: ImageOverride -equals Build.Server.Amd64.VS2019.Open
+          demands: ImageOverride -equals 1es-windows-2019-open
         ${{ if eq(variables['System.TeamProject'], 'internal') }}:
           name: NetCore1ESPool-Internal
-          # Visual Studio Enterprise - contains some stuff, like SQL Server and IIS Express, that we use for testing
-          demands: ImageOverride -equals Build.Server.Amd64.VS2019
+          demands: ImageOverride -equals 1es-windows-2019
```

Comment:
To confirm: The Helix-related images aren't _required_ for official builds @MattGal @mmitche❔

Reply:
correct these are images provide from the people who make us use them, they are more proper than the Build. ones.